### PR TITLE
Added server fetched 'Hello person' quote to my main page.

### PR DESCRIPTION
**Pull request by max9613**
Added server fetched 'Hello person' quote to my main page.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -6,10 +6,11 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getDataMessage()">
```
Comment (735952) [inline portfolio/src/main/webapp/index.html]: Although "get" is not wrong, I think "loadGreetingsContainer" or something like that might be better.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -29,3 +29,14 @@ function getFunFact() {
...
   factContainer.innerText = fact;
 }
+
+/**
+ * Fetches a the message at /data.
+ */
+function getDataMessage() {
+  console.log('Fetching data message.');
+
+  fetch('/data').then(response => response.text()).then((quote) => {
```
Comment (735952) [inline portfolio/src/main/webapp/script.js]: using async and await might yield a slightly more readable code.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("Hello person!");
```
Comment (735952) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: I think you should use your own name instead of "person".

**Review by 735952 — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (max9613) [reply portfolio/src/main/webapp/index.html]: okay I'll make this change

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (max9613) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Okay

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (max9613) [reply portfolio/src/main/webapp/script.js]: Okay I'll try that

**Review by 735952 — APPROVED** [review]

**Pull request merged**